src/main/java/net/minestom/server/network/packet/server/play/ActionBarPacket.java

```diff
@@ -21,4 +26,14 @@ public void write(@NotNull BinaryWriter writer) {
     public int getId() {
         return ServerPacketIdentifier.ACTION_BAR;
     }
+
+    @Override
+    public @NotNull Collection<Component> components() {
+        return Collections.singleton(this.text);
```

Comment:
`Collections.singleton` should be replaced to `List.of` (we currently use `singleton` in some of our types, but this is a mistake)

Reply:
done